File: SozlukJava/app/src/main/java/com/example/sozlukjava/model/SimpleResponse.java
package com.example.sozlukjava.model;


public class SimpleResponse {
    private boolean success;
    private String message;
    private Integer user_id;
    private Integer entry_id;
    private Integer comment_id;

    public SimpleResponse(boolean success, String message, Integer user_id, Integer entry_id, Integer comment_id) {
        this.success = success;
        this.message = message;
        this.user_id = user_id;
        this.entry_id = entry_id;
        this.comment_id = comment_id;
    }

    public SimpleResponse() {}

    public boolean isSuccess() { return success; }
    public void setSuccess(boolean success) { this.success = success; }

    public String getMessage() { return message; }
    public void setMessage(String message) { this.message = message; }

    public Integer getUser_id() { return user_id; }
    public void setUser_id(Integer user_id) { this.user_id = user_id; }

    public Integer getEntry_id() { return entry_id; }
    public void setEntry_id(Integer entry_id) { this.entry_id = entry_id; }

    public Integer getComment_id() { return comment_id; }
    public void setComment_id(Integer comment_id) { this.comment_id = comment_id; }
}

File: sozluk_flutter/lib/model/simple_response.dart
class SimpleResponse {
  final bool success;
  final String? message;
  final int? userId;
  final int? entryId;
  final int? commentId;

  SimpleResponse({
    required this.success,
    this.message,
    this.userId,
    this.entryId,
    this.commentId,
  });

  factory SimpleResponse.fromJson(Map<String, dynamic> json) {
    return SimpleResponse(
      success: json['success'],
      message: json['message'],
      userId: json['user_id'],
      entryId: json['entry_id'],
      commentId: json['comment_id'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'success': success,
      'message': message,
      'user_id': userId,
      'entry_id': entryId,
      'comment_id': commentId,
    };
  }
}
